projects/demo/src/modules/app/classes/ts-file.parser.ts

```diff
@@ -0,0 +1,49 @@
+const CLASS_NAME_REGEXP = /(?<=export class\s)(\w*)/gi;
+
+export class TsFileParser {
+    get className(): string {
+        const [className] = this.rawFileContent.match(CLASS_NAME_REGEXP) || [''];
+
+        return className;
+    }
+
+    set className(newClassName: string) {
+        this.rawFileContent = this.rawFileContent.replace(
+            CLASS_NAME_REGEXP,
+            newClassName,
+        );
+    }
+
+    get isNgModule(): boolean {
+        return this.rawFileContent.includes('@NgModule');
```

Comment:
We might have some examples with both module and component in the same file. Gotta keep in mind not to do that anymore :)

Reply:
```
We might have some examples with both module and component in the same file. Gotta keep in mind not to do that anymore :)
```

maybe rename `get isNgModule` to `get hasNgModules()`?